tbx/core/blocks.py

```diff
@@ -297,6 +298,50 @@ class IconChoice(models.TextChoices):
     WAGTAIL = "wagtail", "wagtail icon"
 
 
+class DivisionSignpostCardValue(blocks.StructValue):
+    def get_heading(self):
+        if override := self.get("heading"):
+            return override
+        return self["page"].title
```

Comment:
Would this be simpler to do in `DivisionSignpostCardBlock.get_context()`?

Reply:
Or even a `{% firstof card.heading card.page.title %}` in the template

Reply:
Thanks, I've tweaked it to use `firstof` instead. I've gotten too used to using `StructValue`!